### DISCUSSION TITLE
0081 XLS-81d: Permissioned DEXes

### DISCUSSION THREAD

**Discussion by mvadari** [XRPLF/XRPL-Standards#229] [XLS Proposals]
## The earlier version can be found by looking at the edit history.

**Comment by Island-Ghost** [comment]
This is fantastic. I hope to be a factoring contributor to this development.

**Comment by XRPSezz** [comment]
Really well thought out practical and comprehensive proposal.

**Comment by mvadari** [comment]
A minor update to this spec was published today, 2024-09-25. Token-gating support was removed, as per the change in [XLS-80d](https://github.com/XRPLF/XRPL-Standards/discussions/228).

**Comment by HigherAndHigher** [comment]
Great, looking forward to the next article.

**Comment by mvadari** [comment]
A minor update to this spec was published today, 2024-12-02, to support offers that can be in a domain _and_ the open DEX.

**Comment by mvadari** [comment]
Per the new [XLS Contributing process](https://github.com/XRPLF/XRPL-Standards/blob/master/CONTRIBUTING.md), it is my opinion that we have reached a "well-refined standard."

As such, I propose that we move this discussion to a file (via https://github.com/XRPLF/XRPL-Standards/pull/256) and work on final changes using additional PRs, for better change-tracking.

Please comment here if you would like to object to moving this spec/discussion forward in the process into a DRAFT spec.

(Note that per the Contributing guidelines, moving a spec into the [DRAFT](https://github.com/XRPLF/XRPL-Standards/blob/master/CONTRIBUTING.md#drafts) state does not mean any kind of endorsement, nor does it mean that this specification will become adopted. It is solely meant as a mechanism to enable better change tracking using PRs.)
